nginx.conf

```
@@ -0,0 +1,28 @@
+events {
+  worker_connections 1024;
+}
+
+http {
+  server {
+    listen 80;
+    server_name 4tunas-ork-rush-28.ya-praktikum.tech;
+
+    location / {
+      return 301 https://$host$request_uri;
+    }
+  }
+
+  server {
+    listen 443 ssl http2;
```

```suggestion
    listen 443 ssl;
    
    http2 on;
```

В логах пишет, что данная запись внутри директивы `listen`  **deprecated**. В [документации](http://nginx.org/en/docs/http/ngx_http_v2_module.html) предлагается прописать отдельной строкой.